core/modules/system/system.install

```diff
@@ -245,9 +245,11 @@ function system_requirements($phase) {
         'directory' => $private_files_directory,
       );
     }
+    $temporary_directory = file_directory_temp();
+    config_set('system.core', 'file_temporary_path', $temporary_directory);
     $htaccess_files['temporary://.htaccess'] = array(
       'title' => $t('Temporary files directory'),
-      'directory' => file_directory_temp(),
+      'directory' => $temporary_directory,
```

Comment:
This has at least one drawback: it only strikes if backdrop_is_apache().

Eventually it's better to do this in phase 'install' and/or 'update'. Not sure about that yet.

Reply:
Please also read https://github.com/backdrop/backdrop-issues/issues/4193#issuecomment-550338846 

So we are back to my first PR?

Reply:
I'm not the one to decide in any way, I just comment possible glitches here. ;-)
Sorry, I'm currently too busy to do more.